doc/source/user_guide/mapdl.rst

```diff
@@ -1097,83 +1097,106 @@ Environment variables
 =====================
 
 There are several PyMAPDL-specific environment variables that can be
-used to control the behavior or launching of PyMAPDL and MAPDL.
+used to control the default behavior of PyMAPDL or launching MAPDL.
+
+It should be mentioned that these environment variables do not have
+priority over the arguments given in the corresponding functions.
+For instance:
+
+.. code-block:: console
+    
+    user@machine:~$ export PYMAPDL_PORT=50052
+    user@machine:~$ python -c "from ansys.mapdl.core import launch_mapdl; mapdl=launch_mapdl(port=60053)"
+
+The preceding command launches an MAPDL instance on the port 60053,
+because the argument ``port`` has priority over the environment
+variable :envvar:`PYMAPDL_PORT`.
+
 These are described in the following table:
```

Comment:
```suggestion
This command launches an MAPDL instance on port 60053
because the ``port`` argument has priority over the :envvar:`PYMAPDL_PORT`
environment variable. The following table describes all arguments.

```